common/vpc_stack.py

```diff
@@ -18,6 +20,10 @@ def __init__(self, scope: Construct, context: str, env: dict, **kwargs) -> None:
                            cidr=env.get(VPC_CIDR_CONTEXT),
                            max_azs=2)
 
+        bucket_arn=f"arn:aws:s3:::{FLOW_LOGS_BUCKET}"
+        bucket=s3.Bucket.from_bucket_attributes(self, id=FLOW_LOGS_BUCKET, bucket_arn=bucket_arn)
+        self.vpc.add_flow_log(f"{stack_id}-FlowLogS3",destination=ec2.FlowLogDestination.to_s3(bucket=bucket))
```

Comment:
I don't think this is going to work because I believe this will require ECS to have access to the central `sagebase-vpc-flow-logs-bucket-bucket-5lvxjv2gp37h` bucket that's in a separate AWS account.   I could be wrong but I think you will need to setup cross account bucket access for this to work.

Reply:
We are already getting flow logs from Synapse prod (as well as many other accounts).